src/Files.App/Views/Layouts/DetailsLayoutPage.xaml

```diff
@@ -931,8 +932,9 @@
 											</ContentPresenter>
 
 											<!--  Loading indicator  -->
-											<uc:Shimmer
+											<wctlabs:Shimmer
 												x:Name="TypeUnknownGlyph"
+												MinWidth="8"
```

Comment:
Does `MinHeight` need to be set as well?

Reply:
Yeah when I tried this change locally, Shimmer was clipped because default style defines the min width 40px instead of 8px which was set in our own Shimmer style.
We can set this globally, do you want me to do so?

Reply:
Please do